Freshli.Cli/nuget.config.example

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <add key="GithubPackages" value="https://nuget.pkg.github.com/corgibytes/index.json" />
+    </packageSources>
+    <packageSourceCredentials>
+        <GithubPackages>
+            <add key="Username" value="GITHUB_USERNAME" />
+            <add key="ClearTextPassword" value="PERSONAL_ACCESS_TOKEN" />
+        </GithubPackages>
+    </packageSourceCredentials>
+</configuration>
```

Comment:
Is this needed when the Freshli repo is made public again?

Reply:
Unfortunately it is - always required from Github Packages